HACKING.md

```diff
@@ -42,6 +42,10 @@ We'd love the help!
 - Our mailing list is snapcraft@lists.ubuntu.com
 - We can also be found on the #snappy IRC channel on Freenode
 
+It would help a lot if you setup a pre-commit hook to run at least the
+static tests. a simple way to set this up is by running:
+
+    ./runtests.sh setup-pre-commit
```

Comment:
El 18/08/16 a las 17:16, Kyle Fazzari escribió:
> In HACKING.md
> <https://github.com/snapcore/snapcraft/pull/741#discussion_r75380873>:
> 
>> @@ -42,6 +42,10 @@ We'd love the help!
>>  - Our mailing list is snapcraft@lists.ubuntu.com
>>  - We can also be found on the #snappy IRC channel on Freenode
>>  
>> +It would help a lot if you setup a pre-commit hook to run at least the
>> +static tests. a simple way to set this up is by running:
>> +
>> +    ./runtests.sh setup-pre-commit
> 
> Hahaha, this is for me, isn't it.

For everyone ;-)